go/client/cmd_chat_send.go

```diff
@@ -21,16 +22,20 @@ import (
 	isatty "github.com/mattn/go-isatty"
 )
 
+const maxEphemeralLifetime = time.Hour * 24 * 7
+const minEphemeralLifetime = time.Second * 5
```

Comment:
cc @malgorithms what did you have in mind for these values?

Reply:
I really am not sure . 30 seconds min and one week max ?

Reply:
5 seconds would be nice

Reply:
Btw Telegram is 1 second to 1 week.

Reply:
@dabura667 note that our times here aren't "time from read" but rather "time from send", so times shorter than about 30 seconds are probably too short to be usable. We're doing it this way because we want ephemeral/exploding messages to work well in groups of people, and a "time from read" rule can be harder to reason about there.